frontend/app/components/user-component/my-badges.js

```diff
@@ -21,6 +21,32 @@ export default Component.extend({
         });
     },
 
+    updateBadgeName() {
```

Comment:
Can ask for the `badge` as the parameter and send it to parent controller to update it as it happens in real time in the store.